packages/database/src/migrations/migrationData/20210728020128-AddFQSSurveySyncGroups.js

```diff
@@ -1,67 +1,283 @@
+const FQS1_ANSWER_MAP = {
+  0: 'Major Issue',
+  50: 'Minor Issue',
+  100: 'Achieved',
+};
+
+const FQS2_ANSWER_MAP = {
+  0: 1,
+  50: 2,
+  100: 3,
+};
+
 export const FQS1_CODE_MAPPING = {
-  lesmis_province: 'FQS1Primary_location/FQS1PrimaryProvince',
-  lesmis_district: 'FQS1Primary_location/FQS1PrimaryDistrict',
-  lesmis_school: 'FQS1Primary_location/FQS1PrimarySchool',
+  lesmis_school: { koboQuestionCode: 'FQS1Primary_location/FQS1PrimarySchool' },
 
-  lesmis_fqs1_stdbuilding: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q1',
-  lesmis_fqs1_office: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q2',
-  lesmis_fqs1_cabinet: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q3',
-  lesmis_fqs1_water: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q4',
-  lesmis_fqs1_latrine: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q5',
-  lesmis_fqs1_board: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q6',
-  lesmis_fqs1_map: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q7',
-  lesmis_fqs1_table: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q8',
-  lesmis_fqs1_chair: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q9',
-  lesmis_fqs1_t_tablechair: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q10',
+  lesmis_fqs1_stdbuilding: {
+    koboQuestionCode: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q1',
+    answerMap: FQS1_ANSWER_MAP,
+  },
+  lesmis_fqs1_stdbuilding_c: { koboQuestionCode: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q1Comment' },
+  lesmis_fqs1_office: {
+    koboQuestionCode: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q2',
+    answerMap: FQS1_ANSWER_MAP,
+  },
+  lesmis_fqs1_office_c: { koboQuestionCode: 'FQS1PrimaryPart1/FQS1PrimaryPart1Q2Comment' },
```

Comment:
These comment questions were missing from the earlier version of the tupaia survey I was looking at, so I've also added them in now